protocol/packages/swap/src/astroport.rs

```diff
@@ -23,37 +33,42 @@ use sdk::{
     },
     cosmwasm_std::{self, Coin as CwCoin, Decimal},
 };
-use serde::{Deserialize, Serialize};
 
 pub type RequestMsg = MsgExecuteContract;
```

Comment:
[here and on Osmosis' impl] just to remind you that 'pub' should be removed once migrate int.tests

Reply:
Will put a TODO for that in this case!